tools/license2rtf.mjs

```diff
@@ -287,19 +286,14 @@ class RtfGenerator extends Stream {
   }
 }
 
-
-const stdin = process.stdin;
-const stdout = process.stdout;
-const lineSplitter = new LineSplitter();
-const paragraphParser = new ParagraphParser();
-const unwrapper = new Unwrapper();
-const rtfGenerator = new RtfGenerator();
-
 stdin.setEncoding('utf-8');
 stdin.resume();
 
-stdin.pipe(lineSplitter);
-lineSplitter.pipe(paragraphParser);
-paragraphParser.pipe(unwrapper);
-unwrapper.pipe(rtfGenerator);
-rtfGenerator.pipe(stdout);
+pipeline(
+  stdin,
+  new LineSplitter(),
+  new ParagraphParser(),
+  new Unwrapper(),
+  new RtfGenerator(),
+  stdout,
+);
```

Comment:
Why is it that we can't wait for the promise to resolve on Windows? Can we add the reason as a comment?

Reply:
we cannot wait for this promise to resolve on MacOS either, it will exit with code 13. 

After some debugging, I found the pipeline will not be resolved. If I put `console.log('pipeline resolved')` after `await pipeline`, the `pipeline resolved` log will not print. But I don't know why the pipeline promise is not resolved.

Reply:
This seems to be a problem somewhere in the original code.
For example, what if we try this in all filters?
```diff
+    this.writable = false;
     this.emit('end');
  }

Reply:
> This seems to be a problem somewhere in the original code. For example, what if we try this in all filters?
> 
> ```diff
> +    this.writable = false;
>      this.emit('end');
>   }
> ```

It works! thank you @LiviaMedeiros